### PR TITLE
A script to setup Windows machine for nativeScript development

### PR DESCRIPTION
**Pull request by teobugslayer**
Setup a fresh Windows machine for developing with NativeScript

Install JDK8, Android SDK, Gradle & Chrome. Setup Android SDK correctly.

### CODE REVIEW

File: setup/native-script.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# install deps with choco
+
+cinst googlechrome
+cinst nodejs.install
+cinst jdk8
+cinst gradle -version 2.3
+cinst android-sdk
+
+# setup android sdk
+echo yes | cmd /c $env:localappdata\Android\android-sdk\tools\android update sdk --filter tools,platform-tools,android-22,build-tools-22.0.1,sys-img-x86-android-22,extra-android-m2repository,extra-google-m2repository,extra-android-support --all --no-ui
```
Comment (rosen-vladimirov) [inline setup/native-script.txt]: we work with android 23 as well :)

**Comment by rosen-vladimirov** [comment]
:+1:  awesome

File: setup/native-script.txt (hunk 1 above)
Comment (teobugslayer) [inline setup/native-script.txt]: I do not understand your remark - what should I change?

Comment (rosen-vladimirov) [inline setup/native-script.txt]: Just noted that you are installing `android-22`, while we also support `android-23` with `1.4.0` of {N} CLI
But anyway, we can keep the current code until we are sure everything is working fine with `android-23`

File: setup/native-script.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+# A boxter script to setup Windows machine for NativeScript development
```
Comment (ikoevska) [inline setup/native-script.txt]: set up, also not sure about the boxter part? If this is a name, this should probably capitalized?

File: setup/native-script.txt [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+# install deps with choco
```
Comment (ikoevska) [inline setup/native-script.txt]: Install dependencies with Chocolatey

File: setup/native-script.txt [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# setup android sdk
+echo yes | cmd /c $env:localappdata\Android\android-sdk\tools\android update sdk --filter tools,platform-tools,android-22,build-tools-22.0.1,sys-img-x86-android-22,extra-android-m2repository,extra-google-m2repository,extra-android-support --all --no-ui
+
+#setup environment
+
+#if (!$env:ANDROID_HOME) { [Environment]::SetEnvironmentVariable("ANDROID_HOME", "$env:localappdata\Android\android-sdk", "User") }
+#if (!$env:JAVA_HOME) { [Environment]::SetEnvironmentVariable("JAVA_HOME", "My Value", "User") }
+#[Environment]::SetEnvironmentVariable("PATH", "$env:PATH;$env:localappdata\Android\android-sdk\tools;$env:localappdata\Android\android-sdk\platform-tools", "User")
+
+write-host -BackgroundColor Black -ForegroundColor Yellow "This script has modified your environment. You need to log off and log on for the changes to take effect."
```
Comment (ikoevska) [inline setup/native-script.txt]: and log back on

File: setup/native-script.txt (hunk 2 above)
Comment (teobugslayer) [inline setup/native-script.txt]: I meant to write boxstarter, I guess I was lacking coffee

**Comment by ErjanGavalji** [comment]
:+1:

**Comment by rosen-vladimirov** [comment]
On my Windows machine:
```
!!ATTENTION!!
The next version of Chocolatey (v0.9.9) will require -y to perform
  behaviors that change state without prompting for confirmation. Start
  using it now in your automated scripts.

  For details on the all new Chocolatey, visit http://bit.ly/new_choco
Chocolatey (v0.9.8.33) is installing 'nodejs.install' and dependencies. By insta
lling you accept the license for 'nodejs.install' and each dependency you are in
stalling.

```

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by teobugslayer** [comment]
@rosen-vladimirov, see the author's explanation [here]( https://github.com/mwrock/boxstarter/issues/89#issuecomment-104542837)

**Pull request merged**